frontend/src/pages/home.rs

```diff
@@ -144,3 +150,22 @@ impl Component for Home {
         };
     }
 }
+// async fn submit_form(client: Arc<Client>, name: String) -> reqwest::Result<Create_response> {
+async fn submit_create_form(client: Arc<Client>, name: String) -> Result<create_response, &'static str> {
+    let mut map = HashMap::new();
+    map.insert("name",name);
+    let response = client.post("http://localhost:9000/game").json(&map).send().await;
```

Comment:
Adresa se s velkou pravděpodobností změní, takhle bys musel při změně procházet celý kód a všude to měnit ručně. Buď to vytáhni na nějaké snadno editovatelné místo, nebo to předávej jako CLI argument - nebudeš muset při změně znovu kompilovat.